templates/node.conf.j2

```diff
@@ -1,51 +1,49 @@
-basedir : "{{ corda_dir_location }}"
-p2pAddress : "{{ corda_host_p2p }}:{{ corda_port_p2p }}"
-webAddress : "0.0.0.0:{{ corda_port_web }}"
-h2port : {{ corda_port_h2 }}
-emailAddress : "{{ corda_admin_email }}"
-myLegalName : "O={{ corda_org }}, L={{ corda_city }}, C={{ corda_country }}"
+p2pAddress = "{{ corda_host_p2p }}:{{ corda_port_p2p }}"
+myLegalName = "O={{ corda_name_org }}, L={{ corda_name_city }}, C={{ corda_name_country }}"
+emailAddress = "{{ corda_admin_email }}"
 
-keyStorePassword : "cordacadevpass"
-trustStorePassword : "trustpass"
 
-dataSourceProperties : {
-    dataSourceClassName : org.h2.jdbcx.JdbcDataSource
-    "dataSource.url" : "jdbc:h2:file:"${baseDirectory}"/persistence"
-    "dataSource.user" : sa
-    "dataSource.password" : ""
-}
+keyStorePassword = {{ corda_password_keystore }}
```

Comment:
`corda_password_keystore` doesn't have any default value and Ansible might fail or walk over that, depending on the Ansible global settings.

It would be better to check if that value has been provided with `assert` Ansible module (see https://docs.ansible.com/ansible/latest/modules/assert_module.html) before using it, or to use `mandatory` (see https://docs.ansible.com/ansible/2.3/playbooks_filters.html#forcing-variables-to-be-defined)

Reply:
What do you suggest? I have no opinion.

Reply:
I would prefer `assert` as it could emit a meaningful message when that variable is not set

Reply:
Actually, there is a bigger problem. It was suggested not to set this variable, but pass it as environment variable. 
Another option is to use  Ansible to pass variable from vault to restart/start Corda. For now I'm going to use assert and log an issue.

Reply:
I used 'assert' and created new issue.